### PR TITLE
Move time module

### PR DESCRIPTION
**Pull request by ThishaniLucas**

### CODE REVIEW

File: time-ballerina/build.gradle [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+task ballerinaBuild {
+    inputs.dir file(project.projectDir)
+    doLast {
+        def originalConfig = ballerinaConfigFile.text
+        def newConfig = ballerinaConfigFile.text.replace("@project.version@", project.version)
+        newConfig = newConfig.replace("@toml.version@", tomlVersion)
+        ballerinaConfigFile.text = newConfig
+
+        def distributionBinPath =  project.projectDir.absolutePath + "/build/target/extracted-distributions/jballerina-tools-zip/jballerina-tools-${ballerinaLangVersion}/bin"
+
```
Comment (Bhashinee) [inline time-ballerina/build.gradle:74]: Shall we enable this config to skip tests when building?

https://github.com/ballerina-platform/module-ballerina-jwt/blob/6f13d4f53f58b65332c2219544130622fad00e96/jwt-ballerina/build.gradle#L114

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,105 @@
+/*
+<<<<<<< HEAD
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+=======
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+>>>>>>> c4b79cec52b638e48caa6eb2b73401af64798ae9
```
Comment (daneshk) [inline build.gradle:6]: Please fix the conflict

File: time-ballerina/build.gradle (hunk 1 above)
Comment (ThishaniLucas) [reply time-ballerina/build.gradle:79]: done

File: time-native/build.gradle [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+plugins {
+    id 'java'
+}
+
+description = 'Ballerina - Time Java Utils'
+
+dependencies {
+    compile group: 'org.ballerinalang', name: 'ballerina-lang', version: "${ballerinaLangVersion}"
```
Comment (Bhashinee) [inline time-native/build.gradle:25]: Do we need this dependency? If not shall we remove it?

**Review by Bhashinee — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**